docs/getting-started/provider-azure.md

```diff
@@ -0,0 +1,216 @@
+<!-- vale gitlab.Substitutions = YES -->
+```shell {label="kube-create-secret",copy-lines="all"}
+kubectl create secret \
+generic azure-secret \
+-n crossplane-system \
+--from-file=creds=./azure-credentials.json
+```
+
+View the secret with `kubectl describe secret`
+
+{{< hint type="note" >}}
+The size may be larger if there are extra blank spaces in your text file.
+{{< /hint >}}
+
+```shell
+kubectl describe secret azure-secret -n crossplane-system
+Name:         azure-secret
+Namespace:    crossplane-system
+Labels:       <none>
+Annotations:  <none>
+
+Type:  Opaque
+
+Data
+====
+creds:  629 bytes
+```
+
+### Create a ProviderConfig
+A `ProviderConfig` customizes the settings of the Azure Provider.  
+
+Apply the {{< hover label="providerconfig" line="2">}}ProviderConfig{{</ hover >}} with the command:
+```yaml {label="providerconfig",copy-lines="all"}
+cat <<EOF | kubectl apply -f -
+apiVersion: azure.upbound.io/v1beta1
+metadata:
+  name: default
+kind: ProviderConfig
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: crossplane-system
+      name: azure-secret
+      key: creds
+EOF
+```
+
+This attaches the Azure credentials, saved as a Kubernetes secret, as a {{< hover label="providerconfig" line="9">}}secretRef{{</ hover>}}.
+
+The {{< hover label="providerconfig" line="11">}}spec.credentials.secretRef.name{{< /hover >}} value is the name of the Kubernetes secret containing the Azure credentials in the {{< hover label="providerconfig" line="10">}}spec.credentials.secretRef.namespace{{< /hover >}}.
+
+
+### Create a managed resource
+A _managed resource_ is anything Crossplane creates and manages outside of the Kubernetes cluster. This creates an Azure Resource group with Crossplane. The Resource group is a _managed resource_.
+
+{{< hint type="tip" >}}
+A resource group is one of the fastest Azure resources to provision.
+{{< /hint >}}
+
+```yaml {label="xr",copy-lines="all"}
+cat <<EOF | kubectl apply -f -
+apiVersion: azure.upbound.io/v1beta1
+kind: ResourceGroup
+metadata:
+  name: example-rg
+spec:
+  forProvider:
+    location: "East US"
+  providerConfigRef:
+    name: default
+EOF
+```
+
+Notice the {{< hover label="xr" line="2">}}apiVersion{{< /hover >}} and {{< hover label="xr" line="3">}}kind{{</hover >}} are from the `Provider's` CRDs.
+
+The {{< hover label="xr" line="5">}}metadata.name{{< /hover >}} value is the name of the created resource group in Azure.  
+This example uses the name `example-rg`.
+
+The {{< hover label="xr" line="8">}}spec.forProvider.location{{< /hover >}} tells Azure which Azure region to use when deploying resources. The region can be any [Azure geography](https://azure.microsoft.com/en-us/explore/global-infrastructure/geographies/) code.
+
+Use `kubectl get resourcegroup` to verify Crossplane created the resource group.
+
+```shell
+kubectl get ResourceGroup
+NAME         READY   SYNCED   EXTERNAL-NAME   AGE
+example-rg   True    True     example-rg      4m58s
```

Comment:
This isn't a valid snippet of Bourne shell.

Reply:
Reasoning has been added to the [contributing guide](https://github.com/crossplane/crossplane/pull/3396/files#diff-93426d953f03b733adbbb574ffd2717d0892773a464c0d301d1b6c9095d67b15R191).

Resolving this conversation (unresolve if you're not satisfied with this)